docs/user/commercial/subscriptions.rst

```diff
@@ -0,0 +1,52 @@
+Read the Docs for Business subscription management
+==================================================
+
+We want to make it easy to manage your billing information.
+You can manage or cancel subscriptions from the :guilabel:`Settings > Subscriptions` page of the :term:`dashboard`.
```

Comment:
`guilabel` is fine to me, but didn't someone have a better solution for documenting UI interactions that involve multiple steps?